docs/functional-components/graphql.md

```diff
@@ -75,6 +80,37 @@ If you use typescript in your project we recomend GraphQL-Code-Generator to gene
 
 https://graphql-code-generator.com
 
+### Instrumentation and Tracking
+
+For instrumentation and tracking we recomend using official OpenTelemetry package
+
+https://www.npmjs.com/package/@opentelemetry/instrumentation-graphql
+
+## Rate Limiting and Query Complexity
```

Comment:
This area is important for the red hat customers. We have been using these packages in production and also as part of the aerogear voyager framework.